docs/README.rst

```diff
@@ -1,12 +1,55 @@
-====
-salt
-====
-
-.. image:: https://travis-ci.com/saltstack-formulas/salt-formula.svg?branch=master
-    :target: https://travis-ci.com/saltstack-formulas
+.. _readme:
+
+salt-formula
+============
+
+|img_travis| |docs| |img_sr|
+
+.. |img_travis| image:: https://travis-ci.com/saltstack-formulas/salt-formula.svg?branch=master
+   :alt: Travis CI Build Status
+   :scale: 100%
+   :target: https://travis-ci.com/saltstack-formulas/salt-formula
+.. |docs| image:: https://readthedocs.org/projects/docs/badge/?version=latest
+   :alt: Documentation Status
+   :scale: 100%
+   :target: https://salt-formula.readthedocs.io/en/latest/?badge=latest
```

Comment:
And coupled with that suggestion above, removing these four lines to do with `|docs|`.